arrus/core/api/devices/us4r/Us4R.h

```diff
@@ -179,6 +179,34 @@ class Us4R : public DeviceWithComponents {
     */
     virtual void setActiveTermination(std::optional<uint16> value) = 0;
 
+    /**
+    * Enables AFE auto offset removal.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void enableAfeAutoOffsetRemoval(void) = 0;
+
+    /**
+    * Disables AFE auto offset removal.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void disableAfeAutoOffsetRemoval(void) = 0;
```

Comment:
What is the default when we dont use the enable/disable AfeAutoOffsetRemoval?

Reply:
Default is disabled, added to comment